pkg/portal/graphql/query.go

```diff
@@ -109,5 +111,55 @@ var query = graphql.NewObject(graphql.ObjectConfig{
 				}).Value, nil
 			},
 		},
+		"activeUserChart": &graphql.Field{
+			Description: "Active users chart dataset",
+			Type:        activeUserChart,
+			Args: relay.NewConnectionArgs(graphql.FieldConfigArgument{
```

Comment:
Do not need to use NewConnectionArgs here. We do not actually use the `after`, `before`, `first` and `last` arguments added by NewConnectionArgs.